doc/manuals/work_queue/examples/wq_python_abstractions.py

```diff
@@ -0,0 +1,37 @@
+#import work_queue
+import work_queue as wq
+
+# Example funtion for the Map Abstraction
+def Map(x):
```

Comment:
This is the right idea, but the naming is confusing b/c you have both `map` and `Map` and only the former does mapping.  :)

Suggest you call these three  `addfunc` `mulfunc` and `maxfunc` or something similar that describes what each function actually does.

Reply:
That makes sense. I was going for "map function is used for map abstraction", But I agree that is definitely less confusing :smile: